src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs

```diff
@@ -80,7 +81,15 @@ public override async ValueTask HandleRpc(ClientPlayer sender, ClientPlayer? tar
                     var tie = reader.ReadBoolean();
 
                     await _eventManager.CallAsync(new MeetingEndedEvent(_game, this));
-
+                    if(playerId != 0xFF){ // make sure a player was voted out
+                        var player = _game.GameNet.GameData.GetPlayerById(playerId);
+                        if(player != null) // fixes a warning
```

Comment:
Linting (Spacing), please refactor to:
```csharp
if (player != null)
``` Also can remove the unnecessary `// fixes a warning` comment.